packages/gatsby/cache-dir/__tests__/loader.js

```diff
@@ -122,36 +108,19 @@ describe(`Loader path prefixing`, () => {
 
   test(`Path prefix present and enabled`, () => {
     global.__PATH_PREFIX__ = `/foo`
-    global.__PREFIX_PATHS__ = true
-    loader.addPagesArray(pagesArray)
-    loader.enqueue(`/foo/about/`)
-
-    expect(loader.___resources()).toEqual([
-      `about.json`,
-      `page-component---src-pages-test-js`,
-    ])
-  })
 
-  test(`Path prefix present but not enabled`, () => {
-    global.__PATH_PREFIX__ = `/foo`
-    delete global.__PREFIX_PATHS__
     loader.addPagesArray(pagesArray)
-
-    // do not enqueue prefixed paths
     loader.enqueue(`/foo/about/`)
-    expect(loader.___resources()).toEqual([])
 
-    // do enqueue unprefixed paths
-    loader.enqueue(`/about/`)
     expect(loader.___resources()).toEqual([
       `about.json`,
       `page-component---src-pages-test-js`,
     ])
   })
 
   test(`Path prefix missing but enabled`, () => {
```

Comment:
And this one to `With default pathPrefix`?

Reply:
Same here

Reply:
👍